Library/Homebrew/cask/lib/hbc/dsl/caveats.rb

```diff
@@ -96,6 +96,12 @@ def eval_caveats(&block)
 
               brew cask install java
           EOS
+        elsif java_version.include?("8")
```

Comment:
This seems potentially unsafe longer-term.

Reply:
Using `elsif java_version.eql?("8")` may be a safer, better choice here.